esp/src/src/Utility.ts

```diff
@@ -1112,6 +1113,41 @@ export function format(labelTpl, obj) {
         .join("\n")
         ;
 }
+const theme = getTheme();
+const { semanticColors } = theme;
+
+export function logColor(level: Level): { background: string, foreground: string } {
+    const colors = {
+        background: "transparent",
+        foreground: "inherit"
+    };
+
+    switch (level) {
+        case Level.debug:
+            colors.background = semanticColors.successBackground;
+            colors.foreground = semanticColors.successIcon;
+            break;
+        case Level.info:
+        case Level.notice:
+            break;
+        case Level.warning:
+            colors.background = semanticColors.warningBackground;
+            colors.foreground = semanticColors.warningIcon;
+            break;
+        case Level.error:
+            colors.background = semanticColors.errorBackground;
+            colors.foreground = semanticColors.errorIcon;
+            break;
+        case Level.critical:
+        case Level.alert:
+        case Level.emergency:
+            colors.background = semanticColors.severeWarningBackground;
+            colors.foreground = semanticColors.severeWarningIcon;
+            break;
+    }
+
+    return colors;
+}
```

Comment:
How well does this work in Dark mode?

Reply:
No difference. Fluent's semantic colors don't change based on the theme it seems, so they're readable in either.